chainer_chemistry/utils/extend.py

```diff
@@ -0,0 +1,66 @@
+from collections import Iterable
+
+import numpy
+import six
+from chainer import cuda
+
+
+def _to_list(a):
+    if isinstance(a, Iterable):
+        a = list(a)
+    else:
+        a = [a]
+    return a
+
+
+def extend_node(node, out_size, axis=-1, value=0):
+    return extend_arrays_to_size(
+        node, out_size=out_size, axis=axis, value=value)
+
+
+def extend_adj(adj, out_size, axis=None, value=0):
+    axis = axis or [-1, -2]
+    return extend_arrays_to_size(
+        adj, out_size=out_size, axis=axis, value=value)
+
+
+def extend_arrays_to_size(arrays, out_size, axis=-1, value=0):
+    """Extend size of `node` array
```

Comment:
Do you think users will use `extend_arrays_to_size` function? Anyway, this docstring  is for `extend_node`.

Reply:
maybe yes, I update docstring.